src/script/run-cbt.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+usage() {
```

Comment:
nit: the help misses `--build-dir` and `--source-dir`.

Reply:
added.